components/analytics/LocationInsights.tsx

```
@@ -0,0 +1,228 @@
+"use client";
+
+import { Eye, MousePointerClick, MapPin, Map, Flag } from "lucide-react";
```

Biome (2.4.7)

[error] Do not shadow the global "Map" property.

(lint/suspicious/noShadowRestrictedNames)